content/en/docs/ops/configuration/traffic-management/dns-proxy/index.md

```diff
@@ -135,10 +141,76 @@ $ kubectl exec deploy/sleep -- curl -sS -v auto.internal
+
+The below example shows how we can make use of DNS proxying as a solution to this problem. A virtual IP address is assigned to the service entries automatically in this case, so that sidecar can clearly distinguish traffic bound for each external TCP service.
+
+1.  Enable DNS automatic address allocation as specified in the [Getting Started](#getting-started) section.
+
+1.  Bring up the client sleep pod as described in the [DNS Capture in Action](#dns-capture-in-action) section.
+
+1.  Deploy the first external sample TCP application:
+
+    {{< text bash >}}
+    $ kubectl create ns external-1
+    $ kubectl -n external-1 apply -f samples/tcp-echo/tcp-echo.yaml
+    {{< /text >}}
+
+1.  Deploy the second external sample TCP application:
+
+    {{< text bash >}}
+    $ kubectl create ns external-2
+    $ kubectl -n external-2 apply -f samples/tcp-echo/tcp-echo.yaml
+    {{< /text >}}
+
+1.  Configure `ServiceEntry` to reach external services:
+
+    {{< text bash >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: networking.istio.io/v1beta1
+    kind: ServiceEntry
+    metadata:
+      name: external-svc-1
+    spec:
+      hosts:
+      - tcp-echo.external-1.svc.cluster.local
+      ports:
+      - name: external-svc-1
+        number: 9000
+        protocol: TCP
+      resolution: DNS
+    ---
+    apiVersion: networking.istio.io/v1beta1
+    kind: ServiceEntry
+    metadata:
+      name: external-svc-2
+    spec:
+      hosts:
+      - tcp-echo.external-2.svc.cluster.local
+      ports:
+      - name: external-svc-2
+        number: 9000
+        protocol: TCP
+      resolution: DNS
+    EOF
+    {{< /text >}}
+
+1.  Verify listeners are configured separately for each service at the client side:
+
+    {{< text bash >}}
+    $ istioctl pc listener deploy/sleep | grep 9000
+    240.240.105.94 9000  ALL                                                                                           Cluster: outbound|9000||tcp-echo.external-2.svc.cluster.local
```

Comment:
This line is very long and hard to read. I wonder if there's a better way to display?

Reply:
changed it, can you check if the new way of printing is better?

Reply:
Yes, I think that's better.

Maybe overkill, but maybe using printf would be even better? For example:
```
awk '{printf "ADDRESS=%s, DESTINATION=%s %s\n", $1, $4, $5}'
```
to display something like this:
```
ADDRESS=240.240.105.94, DESTINATION=Cluster: outbound|9000||tcp-echo.external-2.svc.cluster.local
ADDRESS=240.240.69.138, DESTINATION=Cluster: outbound|9000||tcp-echo.external-1.svc.cluster.local
```

Reply:
Modified